mdk/commands/phpunit.py

```diff
@@ -142,13 +150,26 @@ def run(self, args):
         if testsuite and not testsuite.endswith('_testsuite'):
             testsuite += '_testsuite'
 
+        # Check repeat arg.
+        repeat = args.repeat
+        if repeat:
+            try:
+                # Make sure repeat is an integer and is greater than 1.
+                repeat = int(repeat)
```

Comment:
Use `type=int` to let the parser validate the argument.

Reply:
Thanks, Fred! Good tip! I have fixed this: https://github.com/FMCorz/mdk/pull/185/files#diff-3ddec6e445fe163398be1bc2f84844d4R112